assets/js/atomic/blocks/product-elements/shared/config.tsx

```diff
@@ -28,7 +28,7 @@ const sharedConfig: Omit< BlockConfiguration, 'attributes' | 'title' > = {
 	supports: {
 		html: false,
 	},
-	parent: [ '@woocommerce/all-products', '@woocommerce/single-product' ],
+	ancestors: [ '@woocommerce/all-products', '@woocommerce/single-product' ],
```

Comment:
@gigitux Do we have `ancestors` option? Or I can only find `ancestor` is being used (but [undocumented](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-registration.md)).

I noticed we already defined `ancestor` for the `Add to Cart Button` [here](https://github.com/woocommerce/woocommerce-blocks/pull/7389/files#diff-045de8c6d769a01e12a7e01376e533a9403eceeb56da50862c643cf770153ce7L24-L27). In this case, that setting is controlling the scope of the `Add to Cart Button` block. Did you intend to move that setting to shared config instead?

Reply:
Ops, yeah to correct the key is `ancestor`. 

If I don't rename the key, the block will still be visible only on `'@woocommerce/all-products', '@woocommerce/single-product'` blocks, because we don't override the parent setting.

Reply:
I think we should list `core/post-template` as an ancestor in the shared config eventually, as most of the product elements will be used inside the product query block.

Reply:
As discussed with @danielwrobert, we will move `core/post-template` in the shared config when all the product elements are compatible with the product query block.